itests/kit/blockminer.go

```diff
@@ -256,8 +256,6 @@ func (bm *BlockMiner) MineBlocksMustPost(ctx context.Context, blocktime time.Dur
 }
 
 func (bm *BlockMiner) MineBlocks(ctx context.Context, blocktime time.Duration) {
-	time.Sleep(time.Second)
```

Comment:
This was staggering the time at which block miners started, reducing the chances of having blocks tipsets with multiple blocks.

It was also just unnecessarily slowing down tests.